### DISCUSSION TITLE
[Messenger] 2transports - identify from which transport comes the Message

### DISCUSSION THREAD

**Discussion by etshy** [symfony/symfony#57516] [Q&A]
I have to handle Messages received from a 3rd party system.
On dev and preprod, no problem I listen on the 3rd party preprod queue.
But on prod I have to listen to the preprod queue (for testing purposes) and the prod queue for the ... real usage (different DSN etc.).
I thought about using the `when@prod:` to add a transport on prod but I also need to know from which transport the message comes from to change behaviour depending on it.. 
Should I make two differents handlers to detect the transport difference ?
Or is there a way i don't know yet to identify from which transport the Message is from ?

**Comment by ghost** [comment]
maybe the TransportNamesStamp is what you need

**Reply by etshy** [reply]
Seems like what I need but I'm stuck with 5.4 for now (sorry for not mentioning it) so i'ill have to find something else.
For now I made 2 handlers, one for each transport, and adding a data in some classes to know I'm in "test mode".

**Comment by Elviss89** [comment]
Same issue on a personal Claude Max plan — no Team/Enterprise, no MDM, no admin blocklist. Claude in Chrome / Cowork automation returns "This site is blocked by your organization's policy" on every site.
Confirmed blocked (all in one session):

tomin-family.devbase.pro (my company CRM)
www.pager.co.ua
my.binotel.ua
messenger.smartsender.com
example.com (neutral control — also blocked, so it's literally all sites)

Key facts that rule out a local/org policy:

Personal Max, Org ID 51e470fd-2958-4722-8375-8c14824183df; there is no "Organization settings" / admin panel on my account.
Settings → Claude in Chrome: "Allow extension", blocklist empty.
Settings → Capabilities: network egress ON, Domain allowlist = "All domains".
All these domains open normally in regular Chrome — only Claude's automation is blocked.
It worked earlier in the same session, then started blocking; updating/reinstalling the extension made it worse (went from one domain to all sites).

This is clearly the server-side org_policy:"block" misclassification described here. Please prioritize a fix or a user-facing whitelist — browser automation is currently 100% unusable. Support ticket also opened.
